contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -4387,4 +4386,76 @@ Pointer VirtualAllocEx(HANDLE hProcess, Pointer lpAddress, SIZE_T dwSize,
      * </p>
      */
     boolean VirtualUnlock(Pointer lpAddress, SIZE_T dwSize);
+
+    /**
+     * Sets the user interface language for the current thread.
+     *
+     * <p>
+     * Windows Vista and later: This function cannot clear the thread preferred UI languages list. Your MUI application
+     * should call SetThreadPreferredUILanguages to clear the language list.
+     * </p>
+     * <p>
+     * Windows XP: This function is limited to allowing the operating system to identify and set a value that is safe
+     * to use on the Windows console.
+     * </p>
+     *
+     * @param LangId Language identifier for the user interface language for the thread.
+     *
+     * @return Returns the input language identifier if successful.
+     *         If the input identifier is nonzero, the function returns that value.
+     *         If the language identifier is 0, the function always succeeds and returns
+     *         the identifier of the language that best supports the Windows console.
+     *
+     * <p>
```

Comment:
Please move the remarks above the @param/@return block so that processors don't have to guess where the return entry ends. And add a `<p><strong>Remarks</strong></p>` heading in between.